yarl/_url.py

```diff
@@ -321,22 +322,22 @@ def __hash__(self):
         return ret
 
     def __le__(self, other):
-        if not type(other) is URL:
+        if type(other) is not URL:
```

Comment:
Same as before: if this is desired, it should be a part of a separate PR.

Reply:
This change is not made by me. It was added by you in a force-push activity:

![image](https://github.com/aio-libs/yarl/assets/314607/ad4fc304-b6d8-442c-9c4f-b08495da9c32)

Reply:
@hongquan I didn't touch this branch manually. I only clicked the “Rebase” button on the GitHub UI. This means that the change was already there.
In fact, I specifically remember seeing it earlier. It was added into the very first commit you made: https://github.com/aio-libs/yarl/pull/898/commits/e906337174579cb4d8bac5c1f44ec5ca280a7c3c.

Reply:
Oh, sorry. I was not aware that I made it by accident.